src/ScClient.ts

```diff
@@ -122,21 +128,31 @@ export class ScClient {
     sendData();
   }
 
+  private formSendAnswer(resolve: (arg: any) => void, reject: (arg: any) => void, success: any, errors?: any) {
+    if (errors.length == 0) {
```

Comment:
Looks like in case `errors` is undefined here error may occur then

Reply:
Errors object is object of Error interface or string type. So errors won't occur here, because errors won't null

Reply:
You are right!